README.md

```diff
@@ -79,6 +79,12 @@ You must configure the following parameters in the `./src/ep.config.json` file:
 |`siteImagesUrl`| Optional| String| The path to the site content and marketing images hosted on an external CMS. Set this parameter to the complete URL of the images by replacing the filename and file extension with `%fileName%`. This parameter is populated with the values set in the components when the page is loaded, and uses the assets locally available in the `./src/images/site-images` directory.|
 |`arKit.enable`| Optional| Boolean| Enable elements for ARKit's Quick Look capability to load on a product display page. When `arKit.enable` is enabled, any product images that have hosted ARKit USDZ files are wrapped with an anchor tag referencing the file hosted on an external CMS.|
 |`arKit.skuArImagesUrl`| Optional| String| The path to the USDZ files hosted on an external CMS used for ARKit Quick Look. Set this parameter to the complete URL of the files by replacing the `sku/file-name` with `%sku%`. This value is populated when the page is loaded with values retrieved by Cortex.|
+|`b2b.enable`| Optional| Boolean| The general configuration to enable the B2B e-commerce shopping flow components in the storefront.|
+|`b2b.eamAPI.pathForProxy`| Optional| String| The path to which the [Webpack](https://webpack.js.org/) proxy uses to route Cortex calls from the storefront to the publically hosted EAM service. This value is a URL consisting of a hostname and port of a running Cortex for EAM instance. Leave this field blank to disable proxy.|
```

Comment:
The path to which the: **This sounds wrong ** 

you mean?
The path that the [Webpack](https://webpack.js.org/) proxy uses to route Cortex calls from the storefront to the publicly hosted Account Management service. This value is a URL that consists of the hostname and port of a running Cortex for an Account Management instance. Leave this field blank to disable proxy.

spell error-> publicly